### PR TITLE
Odf to bingham

### PR DESCRIPTION
**Pull request by villalonreina**
Fixed misspellings in the documentation and added references. Added peak_dirs and peak_indices to the BinghamMetric class output.

### CODE REVIEW

File: dipy/direction/bingham.py [hunk 1]
```diff
@@ -540,100 +556,123 @@ def global_voxel_metric(bmetric, bfd):
...
+            peak directions (indexes 12-14);
+            peak indices  (index 15).
         """
         self.model_params = model_params
 
-        self.peak_dirs = model_params[..., 3:6]
+        self.bingham_peak_dirs = model_params[..., 3:6]
+        self.peak_dirs = model_params[..., 12:15]
         self.peak_values = model_params[..., 0]
+        self.peak_indices = model_params[..., 15]
```
Comment (CHrlS98) [inline dipy/direction/bingham.py]: Waiting on these to be reverted before merging (A personal note just to remember where we are at).

Comment (villalonreina) [reply dipy/direction/bingham.py]: @CHrlS98 Will start working on this now. Just one more thing. Have you started working on the comments of @skoudoro?

Comment (CHrlS98) [reply dipy/direction/bingham.py]: No. I want to avoid merge conflicts. I'll address them once I have merged your PR.

Comment (villalonreina) [reply dipy/direction/bingham.py]: Ah okay. I'll let that to you then. Will let you know when I have pushed.

**Comment by villalonreina** [comment]
@CHrlS98 I just pushed to my fork. You can go ahead. Thanks!

**Comment by villalonreina** [comment]
Also, @CHrlS98 and @RafaelNH, One thing I noticed is that we are outputting the same thing twice, `afd` and `peak_values`, which is the `f0`. Should we consolidate them into just one output? And finally, should we have a global `afd`, averaged across all peaks, the same way we have it for all other metrics?

**Comment by CHrlS98** [comment]
@villalonreina Thanks, I'll go ahead with the merge. Also, if `afd` and `peak_values` are the same you should just output one of them and simply write in the doc what they describe.

**Comment by villalonreina** [comment]
@CHrlS98 Yes, I agree. Should I remove one of them (`peak_values` or `afd`), and then create another PR?

**Comment by CHrlS98** [comment]
@villalonreina You can comment on the main PR and I can take care of it along with our other todos.

**Pull request merged**